scriptworker/constants.py

```diff
@@ -205,9 +217,14 @@
                 "schemes": ("https", "ssh", ),
                 "netlocs": ("github.com", ),
                 "path_regexes": (
-                    r"^(?P<path>/mozilla-mobile/focus-android)(/|.git|$)",
-                    r"^(?P<path>/mozilla-mobile/android-components)(/|.git|$)",
-                    r"^(?P<path>/mozilla-mobile/reference-browser)(/|.git|$)",
+                    # XXX We allow Github forks to run staging releases. Please make sure to
+                    # restrict production scopes to the origin repo (with cot_restricted_scopes)!
+                    r"^(?P<path>/({})/android-components)(/|.git|$)".format(
+                        '|'.join(_ALLOWED_MOBILE_GITHUB_OWNERS)
+                    ),
+                    r"^(?P<path>/({})/focus-android)(/|.git|$)".format(
+                        '|'.join(_ALLOWED_MOBILE_GITHUB_OWNERS)
```

Comment:
In an abundance of caution, I think we should run these through `map(re.escape, ...)`.

Reply:
Done